boards/stm32f4discovery/Makefile.include

```diff
@@ -11,6 +11,7 @@ include $(RIOTMAKE)/tools/serial.inc.mk
 
 export DEBUG_ADAPTER ?= stlink
 export STLINK_VERSION ?= 2
```

Comment:
Why here the version is still important and for the others you have moved it?

Reply:
discovery boards use stlink v2 and nucleo boards use v2-1. The default in the stlink.mk.inc file is 2-1, that's why it has to be specified here.

Reply:
F4 doesn't work with ST-LINK 2-1 AFAIK so we must keep this version here.

Reply:
>  for f7 you removed it

Because it was already using 2-1. I left the ones already using v2

Reply:
To summarize the discovery changes:
- f0 and f4: use stlink v2 but a different openocd script => force v2 and the board specific openocd script
- f3: use stlink v2 and the same openocd script as nucleo f3 boards  => force v2 and remove the openocd script so that the common one (in common/st/dist) will be used
- f7: use stlink v2-1 and (nearly) the same openocd script => remove redefinition of v2-1 (useless) and remove the openocd script. Maybe I can keep the previous openocd script but I don't think it hurts the way I did.

Reply:
Ok, I understand now. Thanks for the explanation!